src/main/resources/finalCopy/final_call_logs_dir_local.test

```diff
@@ -1,6 +1,7 @@
 # Tests that a runtime option to save call logs is copying said logs appropriately.
 
 name: final_call_logs_dir_local
+backends: [Local]
```

Comment:
I think we want `Tes` too in that list. I also think that Tes requires a docker ? Now that I'm writing this and ToL: why do we need different WDLs for all those tests ?
I think we could re-use the same and just change the workflow options ?